iscsi/multipath.go

```diff
@@ -8,11 +8,14 @@ import (
 	"os/exec"
 	"strings"
 	"time"
+
+	"k8s.io/klog/v2"
 )
 
 // ExecWithTimeout execute a command with a timeout and returns an error if timeout is excedeed
-func ExecWithTimeout(command string, args []string, timeout time.Duration) ([]byte, error) {
-	debug.Printf("Executing command '%v' with args: '%v'.\n", command, args)
+func ExecWithTimeout(ctx context.Context, command string, args []string, timeout time.Duration) ([]byte, error) {
```

Comment:
`ctx` and `timeout` are redundant. The caller can pass in a context created with `context.WithTimeout` to achieve the same effect - one of the advantages of designing an API which takes a `context` :smile:

Reply:
Using logger pointer now, since this is an internal call.